### PR TITLE
feat: implement welcome page shortcut to reuse last chat for latest info

### PR DESCRIPTION
**Pull request by jaredsavage**
## Note:
Needs copy review for en, fr, and de.

Stuffs classes at end of chat into localstorage. On welcome page, if this is present, and the classes are still available, shows a link to jump directly to the latest results for that class set.

If it's no longer valid, shows text that asks the user to redo the chat.

## Link
Screen Shot 2020-03-16 at 3 58 08 PM

## Expired class set
Screen Shot 2020-03-16 at 3 58 58 PM

### CODE REVIEW

File: src/pages/welcome.tsx [hunk 1]
```diff
@@ -103,6 +118,19 @@ export const WelcomePage: React.FC = () => {
             {t('welcomePage.button')}
           </GetStartedButton>
         </div>
+        <Subtext>
+          {previousRunClasses && (
+            <p>
+              {!previousRunIsValid ? (
```
Comment (vicrep) [inline src/pages/welcome.tsx]: tiny stylistic thing, but would find it clearer if we swapped the ternary so it wasn't based on a negation
```
previousRunIsValid ? <Link ... /> : t('welcomePage.previousRunExpired')
```

File: src/pages/welcome.tsx [hunk 2]
```diff
@@ -52,6 +53,16 @@ const Container = styled.div`
   background-color: ${props => props.theme.colors.background};
 `
 
+const Subtext = styled.h4`
+  color: ${props => props.theme.colors.text};
+  font-size: calc(${props => props.theme.sizes.buttonText} * 0.75);
+  padding: 0 calc(${props => props.theme.sizes.buttonText} * 0.75);
```
Comment (vicrep) [inline src/pages/welcome.tsx]: this could be expressed as 0.75em, no?
```suggestion
  padding: 0 0.75em;
```

File: src/styles.ts [hunk 3]
```diff
@@ -11,6 +11,18 @@ const GlobalStyles = createGlobalStyle`
     font-weight: 300;
   }
 
+  a {
```
Comment (vicrep) [inline src/styles.ts]: why are we adding doing global styles here? is this in scope of the PR?

File: src/styles.ts [hunk 4]
```diff
@@ -11,6 +11,18 @@ const GlobalStyles = createGlobalStyle`
...
+    color: ${props => props.theme.colors.primary};
+    text-decoration: none;
+    transition: opacity ease-in-out 0.15s;
```
Comment (vicrep) [inline src/styles.ts]:
```suggestion
    transition: opacity ease-in-out 150ms;
```

File: src/styles.ts (hunk 3 above)
Comment (jaredsavage) [reply src/styles.ts]: links are unstyled right now, so they're purple / navy / whatever the browser sets them to. The inline link used for this feature is an <a tag.

I think it's reasonable to have a base style for a tags

File: src/regions/ca/i18n/translation.fr.ts [hunk 5]
```diff
@@ -21,7 +21,11 @@ export default {
     title: 'COVID-19: Ce que vous devez savoir',
     description:
       'Obtenez des informations fiables et personnalisées auprès de sources médicales canadiennes crédibles concernant le COVID-19.',
-    button: 'Démarrer'
+    button: 'Démarrer',
+    previousRunLink:
+      'Voir les informations les plus récentes pour votre dernière session',
+    previousRunExpired:
+      'Les réponses de la dernière session ont expiré. Veuillez lancer un nouveau chat pour obtenir les informations les plus récentes.'
```
Comment (vanessajones) [inline src/regions/ca/i18n/translation.fr.ts]: 'Les réponses de votre dernière session ont expiré. Veuillez lancer une nouvelle conversation pour obtenir les informations les plus récentes.'

File: src/regions/ca/i18n/translation.fr.ts [hunk 6]
```diff
@@ -21,7 +21,11 @@ export default {
     title: 'COVID-19: Ce que vous devez savoir',
     description:
       'Obtenez des informations fiables et personnalisées auprès de sources médicales canadiennes crédibles concernant le COVID-19.',
-    button: 'Démarrer'
+    button: 'Démarrer',
+    previousRunLink:
+      'Voir les informations les plus récentes pour votre dernière session',
```
Comment (vanessajones) [inline src/regions/ca/i18n/translation.fr.ts]: 'Voir les informations les plus récentes de votre dernière session',
